Vizzuality/i2i
# Feature/layers icons

Description of the PR

## Testing instructions
Go go data-portal/geospatial-data, click on a country. Select layers in layers tab

![Screenshot 2019-10-29 at 13 28 57](https://user-images.githubusercontent.com/33252015/67767096-30871e80-fa50-11e9-9575-49b8e97e5ee8.png)

analysis icon should appear just in analizable layers

## Pivotal Tracker

https://www.pivotaltracker.com/story/show/169184419
